### ISSUE TITLE
There are unKnow layer type : VideoData in C3D V1.1

### ISSUE THREAD

**Issue by nigegedck** [facebook/C3D#356]
I run the C3D V1.1 finetuning. There was a error: unKnow layer type : VideoData. 
I check the file "train_resnet18_r2.prototxt", the layer named "data",its tpye is VdieoData. And I check the file "caffe.proto", actually, there no layer type named "VideoData"
In C3D v1.0, there are type VIDEO_DATA, I can run the demo.
So, could tell me how to deal the problem and run the C3D V1.1. Thank you very much.

**Comment by dutran** [comment]
@nigegedck We have added VideoData in v1.1. Check it here (https://github.com/facebook/C3D/blob/master/C3D-v1.1/src/caffe/layers/video_data_layer.cpp)

**Comment by nigegedck** [comment]
@dutran Thank you very much. I am so sorry for my late reply. Because I can't use PC until today.
I run the C3D V1.1 finetuning again. But it was also this error:
I0223 19:53:38.084920 19948 layer_factory.hpp:77] Creating layer data
F0223 19:53:38.084947 19948 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: VideoData (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, BlackHole, Concat, ContrastiveLoss, Convolution, Convolution3D, Crop, Data, Deconvolution, Deconvolution3D, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, InfogainLoss, InnerProduct, Input, L2Normalize, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Pooling3D, Power, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile)
*** Check failure stack trace: ***
    @     0x7f14255410fd  google::LogMessage::Fail()
    @     0x7f1425543dd3  google::LogMessage::SendToLog()
    @     0x7f1425540c79  google::LogMessage::Flush()
    @     0x7f142554264e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f1425b751f0  caffe::LayerRegistry<>::CreateLayer()
    @     0x7f1425ce5694  caffe::Net<>::Init()
    @     0x7f1425ce77e5  caffe::Net<>::Net()
    @     0x7f1425d10906  caffe::Solver<>::InitTrainNet()
    @     0x7f1425d11df3  caffe::Solver<>::Init()
    @     0x7f1425d1212f  caffe::Solver<>::Solver()
    @     0x7f1425d0b4b1  caffe::Creator_SGDSolver<>()
    @           0x40e74c  caffe::SolverRegistry<>::CreateSolver()
    @           0x408f58  train()
    @           0x405c47  main
    @     0x7f1424549830  __libc_start_main
    @           0x406699  _start

Could you tell me why. Thank you very much.
